src/main/scala/no/ndla/imageapi/controller/HealthController.scala

```diff
@@ -8,15 +8,49 @@
 
 package no.ndla.imageapi.controller
 
-import org.scalatra.{Ok, ScalatraServlet}
+import no.ndla.imageapi.ImageApiProperties
+import no.ndla.imageapi.model.api.SearchResult
+import org.json4s._
+import org.json4s.native.JsonParser
+import org.scalatra.{ActionResult, InternalServerError, Ok, ScalatraServlet}
+
+import scalaj.http.{Http, HttpResponse}
 
 trait HealthController {
   val healthController: HealthController
 
   class HealthController extends ScalatraServlet {
 
+    implicit val formats = DefaultFormats
+
+    def getImageSearchResponse(url: String): HttpResponse[String] = {
+      Http(url).asString
+    }
+
+    def getImageUrl(response: HttpResponse[String]): (String, Long) = {
+      val json = JsonParser.parse(response.body).extract[SearchResult]
+      (json.results.head.metaUrl,json.totalCount)
+    }
+
+    def getReturnCode(imageResponse: HttpResponse[String]): ActionResult = {
+      imageResponse.code match {
+        case 200 => Ok()
+        case _ => InternalServerError()
+      }
+    }
+
     get("/") {
-      Ok()
+      val searchResponse = getImageSearchResponse(
+        s"http://0.0.0.0:${ImageApiProperties.ApplicationPort}${ImageApiProperties.ImageApiBasePath}/v2/images/")
+      val (imageUrl,totalCount) = getImageUrl(searchResponse)
+
+      if(totalCount == 0)
+        Ok()
```

Comment:
Bra! Kom på i går kveld at vi må ta høyde for at det ikke finnes bilder i APIet.
Når det er sakt er har vel ikke denne if-testen noe effekt siden den ikke returnerer `Ok()` fra controlleren.